bestfilament_abs_skyblue.xml.fdm_material

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>Best Filament</brand>
+      <material>ABS</material>
+      <color>SkyBlue</color>
+      <label>BF ABS SkyBlue</label>
+    </name>
+    <adhesion_info>Обязательно использовать клей и желательно закрытый принтер.</adhesion_info>

```

Comment:
Our profiles are not translated. Could you write the adhesion info and description in English? That captures most of Cura's user base.

Reply:
Translated into english